client/app/src/main/java/com/example/criengine/Fragments/MyBooksListFilterFragment.java

```diff
@@ -119,10 +121,32 @@ public void onClick(DialogInterface dialogInterface, int i) {
      */
     @Override
     public void onCheckedChanged(CompoundButton checkBox, boolean isChecked) {
-        if (isChecked) {
-            filterStatus.add(checkBox.getText().toString().toLowerCase());
+        switch (checkBox.getId()) {
+            case R.id.checkbox_available_filter:
+                modifyStatusArray(checkBox, "available");
+                break;
+            case R.id.checkbox_requests_filter:
+                modifyStatusArray(checkBox, "requested");
```

Comment:
i feel like these should match the checkbox texts, maybe make it some sort of constant? static final or something?

Reply:
The entire point of this was so that we can have different check box text to make it more descriptive lmao.
The part you're pointing at is the status of the book which is given to us from the requirements.

Reply:
ah ic, alright then